python/ray/_raylet.pyx

```diff
@@ -351,6 +351,13 @@ cdef execute_task(
     # Automatically restrict the GPUs available to this task.
     ray.utils.set_cuda_visible_devices(ray.get_gpu_ids())
 
+    # Helper method used to exit current asyncio actor
+    def exit_current_actor_if_asyncio():
```

Comment:
Could you add a brief comment that this works because cancellation is only implemented for **non-actor** tasks?